hebiros_gazebo_plugin/include/hebiros_gazebo_joint.cpp

```diff
@@ -2,14 +2,17 @@
 #include <hebiros_gazebo_joint.h>
 
 
+// TODO: support X8, too
 HebirosGazeboJoint::HebirosGazeboJoint(std::string name,
-  std::shared_ptr<ros::NodeHandle> n) {
+  std::shared_ptr<ros::NodeHandle> n)
+  : temp(hebiros::sim::TemperatureModel::createX5()) {
 
   this->name = name;
 
   this->imu_subscriber = n->subscribe<sensor_msgs::Imu>(
     "/hebiros_gazebo_plugin/imu/"+name,
     100, boost::bind(&HebirosGazeboJoint::SubIMU, this, _1));
```

Comment:
is `boost::bind` more commonly used in ROS as opposed to `std::bind`?

Reply:
Yeah, although it might be primarily historical.  I want to change a lot of this type of thing in the existing code.